plugins/woocommerce-blocks/assets/js/blocks/add-to-cart-with-options/edit.tsx

```diff
@@ -42,6 +55,9 @@ const INNER_BLOCKS_TEMPLATE: InnerBlockTemplate[] = [
 	],
 ];
 
+// Get product type options.
+const productTypes = getProductTypeOptions();
```

Comment:
Out of curiosity, why define this constant outside of the component? If the component will never be rendered, we are defining this constant unnecessarily.

Reply:
This is a good and fair point. I'd move it into the render component, but considering this is a temporary implementation, I prefer handling it in [a follow-up](https://github.com/woocommerce/woocommerce/pull/53366).